Elastic-leadership-guide.md

```
@@ -0,0 +1,282 @@
+# Elastic Leadership（エラスティックリーダーシップ）実践ガイド
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)